doc/cephfs/fs-volumes.rst

```diff
@@ -134,6 +113,33 @@ The output format is json and contains fields as follows.
 * used_size: Current used size of the CephFS volume in bytes
 * pending_subvolume_deletions: Number of subvolumes pending deletion
 
+Sample output of volume info command::
+
+  $ ceph fs volume info vol_name
```

Comment:
Maybe you should also include the flag `--human_readable` in this example?

Reply:
I don't think we should give the sample output with the optional flag. Rather, we can have one more output example with the flag, but that looks quite redundant/odd. And for others existing command with optional flag, we aren't adding the output example with the optional flags. WDYT @vshankar @ajarr @kotreshhr

Reply:
I don't have a strong opinion here but it makes sense not to include the output for every optional argument of the command.